### Biomasa: Cantidad y energía

El propietario de una explotación agraria de la provincia de Sevilla (España), posee una superficie
agrícola tanto de secano como de regadío, así como una cabaña ganadera de bovino y porcino. Debido al
coste de tratamiento de los residuos biomásicos que se producen en su explotación, desea analizar las
posibilidades de valorización energética de los mismos.

Dicha explotación está formada por cultivos herbáceos y leñosos. Dentro del primer grupo, dispone de
35 hectáreas de cebada y 21 hectáreas de trigo duro en secano. En cuanto al segundo, dispone de 75
hectáreas de naranjos en regadío, 10 hectáreas de almendros y 43 hectáreas de olivo en condiciones de
secano. Asimismo, la explotación cuenta con varias naves de vacuno de cebo, que albergan un total de 120
cabezas.

### Se pide:

1. Cuantificar la cantidad de biomasa sólida residual que se genera anualmente en la explotación.
2. Determinar la energía primaria en la biomasa residual agrícola y ganadera generada.
3. Si la biomasa sólida se emplea para la generación térmica, y sustituye al consumo de gasóleo C o de calefacción, el cual tiene un PCI de 10 100 kcal/kg y una densidad de 0,85 kg/l, se desea conocer el ahorro económico que supondría la utilización de biomasa si el precio del litro de gasóleo se pagara a 1,1 €/l.
4. Si la biomasa gaseosa se empleara en un motor para la generación eléctrica, el cual tiene un rendimiento del 39 %, cuantificar la cantidad anual de kWh vertidos a la red.

### Datos:

**Factores de generación de residuos:**  
Cereales (cebada y trigo): 1,2 t/ha.  
Cítricos: 1,6 t/ha.  
Frutos secos: 1,5 t/ha.  
Olivo: 1,7 t/ha.  
1 vaca = 4,80 kg materia orgánica/día = 0,27 m3/kg materia orgánica.

**Poderes caloríficos inferior:**  
Cereales: 2900 kcal/kg.  
Cítricos y frutos secos: 3300 kcal/kg.  
Olivo: 3120 kcal/kg.  
Biogás: 5500 kcal/m3.  
1 kWh = 860 kcal.

### 1. Estimación de biomasa residual

Tenemos los siguientes datos generados anualmente por rubro:

| Tipo rubro | Rubro | Extensión/cant | Factor gen. resid. |
| ---------- | ----- | -------------- | ----------------- |
| Herbáceo | Cebada | 35 ha | 1,2 t/ha |
| Herbáceo | Trigo | 21 ha | 1,2 t/ha |
| Leñoso | Naranjo | 75 ha | 1,6 t/ha |
| Leñoso | Almendro | 10 ha | 1,5 t/ha |
| Leñoso | Olivo | 43 ha | 1,7 t/ha |
| Vacuno | Cebo | 120 cab | 4,8 kg MO/vaca.dia |

Estimamos la cantidad de biomasa residual mediante el producto de las extensiones de cultivo por el factor de generación de residuos de cada uno.

In [1]:
#Extensiones cultivos (ha)
ext_ceb = 35
ext_trig = 21
ext_nar = 75
ext_alm = 10
ext_oli = 43
#Cantidad ganado
cant_vac = 120

#Factores de generación de residuos (t/ha)
FG_ceb = 1.2
FG_trig = 1.2
FG_nar = 1.6
FG_alm = 1.5
FG_oli = 1.7
#Factor de generación de residuos de ganado (kg/vaca.dia)
FG_vac = 4.8

#Biomasa residual (t/año)
BR_ceb = ext_ceb * FG_ceb
print("Biomasa residual de cebada: %.1f t/año" % BR_ceb)
BR_trig = ext_trig * FG_trig
print("Biomasa residual de trigo: %.1f t/año" % BR_trig)
BR_nar = ext_nar * FG_nar
print("Biomasa residual de naranjo: %.1f t/año" % BR_nar)
BR_alm = ext_alm * FG_alm
print("Biomasa residual de almendro: %.1f t/año" % BR_alm)
BR_oli = ext_oli * FG_oli
print("Biomasa residual de olivo: %.1f t/año" % BR_oli)
BR_vac = cant_vac * FG_vac * 365 / 1000
print("Biomasa residual de cebo: %.1f t/año" % BR_vac)

Biomasa residual de cebada: 42.0 t/año
Biomasa residual de trigo: 25.2 t/año
Biomasa residual de naranjo: 120.0 t/año
Biomasa residual de almendro: 15.0 t/año
Biomasa residual de olivo: 73.1 t/año
Biomasa residual de cebo: 210.2 t/año


In [3]:
BR_vegetal = BR_ceb + BR_trig + BR_nar + BR_alm + BR_oli
print("Biomasa residual total vegetal: %.1f t/año" % BR_vegetal)
BR_total = BR_vegetal + BR_vac
print("Biomasa residual total: %.1f t/año" % BR_total)

Biomasa residual total vegetal: 275.3 t/año
Biomasa residual total: 485.5 t/año


La cantidad anual de biomasa residual por rubro la podemos resumir complemnetando la tabla anterior:

| Tipo rubro | Rubro | Extens/cant | Fact. gen. res. | **B.Res. (t/año)** |
| ---------- | ----- | ----------- | --------------- | ------------------ |
| Herbáceo | Cebada | 35 ha | 1,2 t/ha | **42.0** |
| Herbáceo | Trigo | 21 ha | 1,2 t/ha | **25.2** |
| Leñoso | Naranjo | 75 ha | 1,6 t/ha | **120.0** |
| Leñoso | Almendro | 10 ha | 1,5 t/ha | **15.0** |
| Leñoso | Olivo | 43 ha | 1,7 t/ha | **73.1** |
| Vacuno | Cebo | 120 cab | 4,8 kg MO/vaca.dia | **210.2** |

De los cuales el total de biomasa residual vegetal es **275,3 toneladas/año** y el total incluyendo la de origen animal es **485,5 toneladas/año**.

## 2. Energía primaria de biomasa residual

El cálculo de energía primaria para la biomasa residual se obtiene del producto de la biomasa residual estimada en el enunciado anterior, escalada a kg, por el poder calorífico inferior (PCI) suministrado en la introducción del caso en kcal/kg:

| Tipo rubro | Rubro | B.Res. (t/año) | PCI (kcal/kg) |
| ---------- | ----- | -------------- | ------------ |
| Herbáceo | Cebada | 42.0 | 2900 |
| Herbáceo | Trigo | 25.2 | 2900 |
| Leñoso | Naranjo | 120.0 | 3300 |
| Leñoso | Almendro | 15.0 | 3300 |
| Leñoso | Olivo | 73.1 | 3120 |
| Vacuno | Cebo | 210.2 | 5500 (1) |

(1) Biogas, kcal/m3. Para el caso del ganado vacuno tenemos como dato que 4,8 kg de materia orgánica por día equivalen a 0,27 m3/kg. Dato necesario para escalar la generacíon diaria a m3/día y así estimar la energía asociada a la cantidad de biomasa y PCI el biogas asociado.

In [4]:
#PCI #kcal/kg
PCI_ceb_tri = 2900
PCI_nar_alm = 3300
PCI_oli = 3120
PCI_vac = 5500 #kcal/m3

#Biomasa ganado en m3/año
BR_vac_m3 = cant_vac * FG_vac * 0.27 * 365

#Energia primaria kcal
EP_ceb = BR_ceb*1000 * PCI_ceb_tri
print("Energía primaria cebada: %.0f kcal/año" % EP_ceb)
EP_trig = BR_trig*1000 * PCI_ceb_tri
print("Energía primaria trigo: %.0f kcal/año" % EP_trig)
EP_nar = BR_nar*1000 * PCI_nar_alm
print("Energía primaria naranja: %.0f kcal/año" % EP_nar)
EP_alm = BR_alm*1000 * PCI_nar_alm
print("Energía primaria almendro: %.0f kcal/año" % EP_alm)
EP_oli = BR_oli*1000 * PCI_oli
print("Energía primaria olivo: %.0f kcal/año" % EP_oli)
EP_vac = BR_vac_m3 * PCI_vac
print("Energía primaria ganado: %.0f kcal/año" % EP_vac)

Energía primaria cebada: 121800000 kcal/año
Energía primaria trigo: 73080000 kcal/año
Energía primaria naranja: 396000000 kcal/año
Energía primaria almendro: 49500000 kcal/año
Energía primaria olivo: 228072000 kcal/año
Energía primaria ganado: 312206400 kcal/año


In [5]:
EP_vegetal = EP_ceb + EP_trig + EP_nar + EP_alm + EP_oli
print("Energía primaria total vegetal: %.0f kcal/año" % EP_vegetal)
EP_total = EP_vegetal + EP_vac
print("Energía primaria total: %.0f kcal/año" % EP_total)

Energía primaria total vegetal: 868452000 kcal/año
Energía primaria total: 1180658400 kcal/año


La tabla anterior con los PCI la podemos actualizar incorporando los valores obtenidos de energía primaria:

| Tipo rubro | Rubro | B.Res. (t/año) | PCI (kcal/kg) | Ener. prim (kcal/año) |
| ---------- | ----- | -------------- | ------------- | ---------------------- |
| Herbáceo | Cebada | 42,0 | 2.900 | **121.800.000** |
| Herbáceo | Trigo | 25,2 | 2.900 | **73.080.000** |
| Leñoso | Naranjo | 120,0 | 3.300 | **396.000.000** |
| Leñoso | Almendro | 15,0 | 3.300 | **49.500.000** |
| Leñoso | Olivo | 73,1 | 3.120 | **228.072.000** |
| Vacuno | Cebo | 210,2 | 5.500 | **312.206.400** |

La energía primaria total de los residuos vegetales es de **868.452.000 kcal/año** y el total incluyendo la asociada a los residuos ganaderos es de **1.180.658.400 kcal/año**.

### 3. Ahorro económico por sustitución de gasóleo

Se nos pide estimar el ahorro asociado a la sustitución de gasóleo por biomasa sólida, de los vegetales, generada. Como datos tenemos que el gasóleo tiene un PCI de 10.100 kcal/kg, una densidad de 0,85 kg/l y un precio de 1,1 €/l.

Estimamos que energía primaria anual asociada a los residuos vegetales es de 868.452.000 kcal/año. La masa anual de gasóleo equivalente se obtiene al dividir este valor entre el PCI del gasóleo. Su conversión a volumen la obtenemos al dividir la masa del gasóleo entre su densidad (0,85 kg/l):

In [7]:
#PCI gasóleo kcal/kg
PCI_gasol = 10100
#Densidad gasóleo kg/l
d_gasol = 0.85
#Masa equivalente gasóleo kg/año
M_gasol = EP_vegetal / PCI_gasol
print("Masa anual equivalente de gasóleo: %.0f kg/año" % M_gasol)
V_gasol = M_gasol / d_gasol
print("Volumen anual equivalente de gasóleo: %.0f l/año" % V_gasol)

Masa anual equivalente de gasóleo: 85985 kg/año
Volumen anual equivalente de gasóleo: 101159 l/año


In [8]:
#Precio de gasóleo por litro
Precio_gasol = 1.1
#Gasto anual de gasóleo (euros)
Gasto_gasol = Precio_gasol * V_gasol
print("Gasto anual equivalente de gasóleo %.0f euros/año" % Gasto_gasol)

Gasto anual equivalente de gasóleo 111275 euros/año


En litros esa masa de gasóleo equivale a 101.159 litros. A un costo promedio de 1,1 €/l, el gasto anual que nos estaríamos ahorrando por aprovechar la biomasa generada en vez de comprar gasóleo sería **111.275 €**.

### 4. Generación eléctrica de biogas producido

Se nos pide estimar cuanta energía eléctrica en KWh podemos generar a partir del biogas derivado de la biomasa animal producida en un año. Como dato se nos dice que el generador eléctrico tiene un rendimiento de 39%. Además sabemos que 1 KWh = 860 Kcal. La energía primaria generada por el biogas es de 312.206.400 kcal por año.

In [12]:
Rendim = 0.39
#Energía eléctrica generada, KWh
Gen_ele = (EP_vac / 860) * Rendim
print("Energía eléctrica generable a partir de biogas con rendimiento 0.39: %.0f KWh" % Gen_ele)

Energía eléctrica generable a partir de biogas con rendimiento 0.39: 141582 KWh


A la red eléctrica local se le pueden inyectar **141.582 KWh** a partir del biogas generado por los residuos del ganado.